src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -46,9 +48,24 @@ export const selectUtxosFee = async ({
 	};
 };
 
-export const sendBtc = async ({ progress, ...rest }: SendBtcParams): Promise<void> => {
+export const sendBtc = async ({
+	progress,
+	utxosFee,
+	network,
+	source,
+	identity,
+	...rest
+}: SendBtcParams): Promise<void> => {
 	// TODO: use txid returned by this method to register it as a pending transaction in BE
-	await send({ progress, ...rest });
+	const { txid } = await send({ progress, utxosFee, network, identity, ...rest });
+
+	await addPendingBtcTransaction({
```

Comment:
shall this be before or after the `RELOAD` step? The previous step is `SEND`, that theoretically is finished at this point, right?

Reply:
I think we can do it after RELOAD to avoid waiting for the call to be completed. Updated.